python/ray/data/impl/pipeline_executor.py

```diff
@@ -26,11 +24,13 @@ def pipeline_stage(fn: Callable[[], Dataset[T]],
 class PipelineExecutor:
     def __init__(self, pipeline: "DatasetPipeline[T]"):
         self._pipeline: "DatasetPipeline[T]" = pipeline
-        self._stages: List[ObjectRef[Dataset[
+        self._stages: List[concurrent.futures.Futre[Dataset[
```

Comment:
Sorry for the typo, my bad! 😬 
```suggestion
        self._stages: List[concurrent.futures.Future[Dataset[
```

Reply:
Haha thank you!